src/Gaufrette/Util/Size.php

```diff
@@ -41,10 +33,8 @@ public static function fromFile($filename)
      * Returns the size in bytes from the given resource.
      *
      * @param resource $handle
-     *
-     * @return string
      */
-    public static function fromResource($handle)
+    public static function fromResource($handle): int
```

Comment:
I don't think this one is right!

Reply:
PHP doc specified int
https://www.php.net/manual/en/function.mb-strlen.php

Reply:
I don't see `mb_strlen` used in this function